outbound.go

```diff
@@ -332,13 +307,22 @@ func cloneTags(tags map[string]string) map[string]string {
 // doneReading shuts down the message exchange for this call.
 // For outgoing calls, the last message is reading the call response.
 func (response *OutboundCallResponse) doneReading(unexpected error) {
-	now := response.GetTime()
-	response.AddAnnotationAt(AnnotationKeyClientReceive, now)
-	response.Report()
+	now := response.timeNow()
 
 	isSuccess := unexpected == nil && !response.ApplicationError()
 	lastAttempt := isSuccess || !response.requestState.HasRetries(unexpected)
 
+	// TODO how should this work with retries?
```

Comment:
That would be hard, because at this low level we don't have access to the headers. I think a better alternative (I believe Finagle does it too) is to add events to the current span, which will show on the timeline as dots (red dots specifically, if we mark them as errors).